k8s/demo/common-overlay/ccd/kustomization.yaml

```diff
@@ -22,7 +21,6 @@ patchesStrategicMerge:
 - ../../../namespaces/ccd/ccd-data-store-api/demo.yaml
 - ../../../namespaces/ccd/ccd-definition-store-api/demo.yaml
```

Comment:
please remove ccd-case-disposer from demo common-overlay as it has been added to the demo cluster ovelays

Reply:
Done, thanks @RababRabab